source/extensions/omni.isaac.lab_tasks/test/test_environments.py

```diff
@@ -38,14 +39,17 @@ def setUpClass(cls):
                 cls.registered_tasks.append(task_spec.id)
         # sort environments by name
         cls.registered_tasks.sort()
-        # print all existing task names
-        print(">>> All registered environments:", cls.registered_tasks)
+
+        # this flag is necessary to prevent a bug where the simulation gets stuck randomly when running the
+        # test on many environments.
+        carb_settings_iface = carb.settings.get_settings()
+        carb_settings_iface.set_bool("/physics/cooking/ujitsoCollisionCooking", False)
```

Comment:
Is this a setting we should set globally?

Reply:
I think David added it just for environment tests to prevent timeouts because of it. Not sure what it's general utility is.